prime-router/metadata/hl7_mapping/STLTs/CO/CO.yml

```diff
@@ -5,10 +5,12 @@ elements:
   #########################
   # ReportStream specific #
   #########################
-  - name: sending-application-namespace-id
+  - name: sending-application-namespace-id-no-endpoint
+    condition: true
     value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
 
-  - name: sending-application-universal-id
+  - name: sending-application-universal-id-no-endpoint
+    condition: true
     value: [ '"2.16.840.1.114222.4.1.237821"' ]
 
```

Comment:
I don't know if we went over this on our calls or if this is on the videos. How are these files used? Is the purpose of these files to have specific rules for different states and how did you know you had to update this file?

Reply:
Yes, it is part of the training on the FHIR to HL7 piece.  We use schema extensions to define a STLT specific schema that can be used to customize the data.  The schema is then specified in the settings for the receiver